.github/workflows/push.yaml

```diff
@@ -211,3 +211,10 @@ jobs:
           allowed-skips: ${{ toJSON(needs) }}
           jobs: ${{ toJSON(needs) }}
 
+  check-rust-windows:
+    runs-on: windows-latest
```

Comment:
can you add an `if` here:

```
    if: ${{ needs.changes.outputs.rust == 'true' }}
```

Reply:
Just makes sure that our typescript-only changes don't take time to build all of rust on windows

Reply:
Makes sense, great catch! It's in there now.